vllm/v1/metrics/wrappers.py

```diff
@@ -0,0 +1,126 @@
+# SPDX-License-Identifier: Apache-2.0
+import time
+from typing import Optional, Union, cast
+
+import prometheus_client
+
+from vllm.config import VllmConfig
+from vllm.v1.metrics.loggers import PrometheusMetrics
+from vllm.v1.spec_decode.metrics import SpecDecodingProm
+
+try:
+    from ray.util import metrics as ray_metrics
+except ImportError:
+    ray_metrics = None
+
+
+class _RayGaugeWrapper:
+    """Wraps around ray.util.metrics.Gauge to provide same API as
+    prometheus_client.Gauge"""
+
+    def __init__(self,
+                 name: str,
+                 documentation: str = "",
+                 labelnames: Optional[list[str]] = None,
+                 multiprocess_mode: str = ""):
+        del multiprocess_mode
```

Comment:
What's going on here? Let's not include multiprocess mode stuff since we're not using it